shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -756,13 +761,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
     } else if (hasToggledState) {
       result.setChecked(semanticsNode.hasFlag(Flag.IS_TOGGLED));
       result.setClassName("android.widget.Switch");
-      result.setContentDescription(semanticsNode.getValueLabelHint());
-    } else if (!semanticsNode.hasFlag(Flag.SCOPES_ROUTE)) {
```

Comment:
Does this change now make SemanticsNodes that are scoping a route and have a label focusable?

Reply:
I'll update the condition so we don't apply it to those nodes, thanks for jogging my memory a bit on the context